ambry-store/src/test/java/com.github.ambry.store/StoreMetricsTest.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class StoreMetricsTest {
```

Comment:
another reason to avoid business logic in metrics classes is this - we don't want to have to write tests for metrics classes.